scubagoggles/reporter/reporter.py

```diff
@@ -393,6 +393,7 @@ def _get_failed_prereqs(self, test: dict) -> set:
             # prerequisites section. For now, to avoid a breaking change,
             # any test without a prerequisites defined gets a free pass.
             prereqs = {}
+            raise RuntimeError(f'No prerequisites found for {self.product}')
```

Comment:
Could you also remove the TODO comment above?

Reply:
Also Pylint is right about this. It does have a `_product`, but it would actually be better to refer to the specific test that has a problem.
```suggestion
            raise RuntimeError(f'No prerequisites found for {test["PolicyId"]}')
```